Bottom: 087abbb5ac7551aca4e8708176f92a412ac0e2a7
Top:    abbc17960483b94bfa0130d2e95d0b2a051f4bc9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-06 01:47:12 +0100

Make --enable-gl default when building the GTK HID


---

diff --git a/configure.ac b/configure.ac
index 2637495..94c6779 100644
--- a/configure.ac
+++ b/configure.ac
@@ -393,8 +393,12 @@ fi
 AC_MSG_CHECKING([for whether to use GL drawing])
 AC_ARG_ENABLE([gl],
 [  --enable-gl           Enable GL drawing (with GTK HID)],
-[],[enable_gl=no])
-
+[],[
+	case " $with_gui " in
+		*\ gtk\ *) enable_gl=yes;;
+		* ) enable_gl=no;;
+	esac
+])
 AC_MSG_RESULT([$enable_gl])
 AM_CONDITIONAL(USE_GL, test x$enable_gl = xyes)
